webknossos/webknossos/client/_generated/api/datastore/data_layer_folder_contents.py

```diff
@@ -0,0 +1,92 @@
+from typing import Any, Dict, Union
+
+import httpx
+
+from ...client import Client
+from ...types import UNSET, Response, Unset
+
+
+def _get_kwargs(
+    organization_name: str,
+    data_set_name: str,
+    data_layer_name: str,
+    *,
+    client: Client,
+    token: Union[Unset, None, str] = UNSET,
+) -> Dict[str, Any]:
+    url = "{}/data/zarr/{organizationName}/{dataSetName}/{dataLayerName}".format(
```

Comment:
These zarr route functions are used anywhere, right?

Reply:
I guess you meant `aren't used anywhere else`? Exactly, that's the only point where the zarr url is constructed.

Reply:
I'd keep them, since they're auto-generated. It would be effort to remove them atm (and keep them removed).